swagger.yaml

```diff
@@ -1974,6 +2193,48 @@ components:
               items:
                 $ref: "#/components/schemas/MetaformScript"
 
+    MetaformVersionType:
+      type: string
+      enum:
+        - ARCHIVED
+        - DRAFT
+
+    MetaformVersion:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        metaformId:
+          type: string
+          format: uuid
+        type:
+          $ref: "#/components/schemas/MetaformVersionType"
+        sections:
```

Comment:
Instead of sections, it should contain whole form as a raw JSON. The form JSON includes all settings for the form, so that would make it more useful and also more robust

Reply:
the field should also be required